client/src/components/Footer/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { Container } from './style';
+import { navRoutes } from '../../constants';
+
+import * as T from '../../components/Typography';
+const Footer = () => {
+  return (
+    <Container>
+      <T.Body16 mb="5">
+        <T.Link to={navRoutes.EXTERNAL.TERMS_OF_USE}>Terms of use</T.Link>
+        <br />
```

Comment:
the ```br``` tag is not semantic and it preferred to use CSS style instead of giving the Link display block 
```suggestion
        <T.Link 
          to={navRoutes.EXTERNAL.TERMS_OF_USE}
          style={{ display: 'block' }}
         >
         Terms of use</T.Link>
       
```

or you can select the a tags from the Container style to give them display block